src/scale_audio_rating.py

```diff
@@ -1,411 +1,411 @@
+                    self.container, permission='read', expiry=datetime.datetime(2019, 10, 30, 12, 30), start=datetime.datetime.now())
+            await self.retrieve_contents(files)
+        elif self._account_type == 'BlobStore':
+            blobs = self.store_service.list_blobs(
+                self.container, self.clips_path)
+            await self.retrieve_contents(blobs)
+
+    def make_clip_url(self, filename):
+        if self._account_type == 'FileStore':
+            source_url = self.store_service.make_file_url(
+                self.container, self.clips_path, filename, sas_token=self._SAS_token)
+        elif self._account_type == 'BlobStore':
+            source_url = self.store_service.make_blob_url(
+                self.container, filename)
+        return source_url
+
+
+class GoldSamplesInStore(ClipsInAzureStorageAccount):
+    def __init__(self, config, alg):
+        super().__init__(config, alg)
+        self._SAS_token = ''
+
+    async def get_dataframe(self):
+        clips = await self.clip_names
+        df = pd.DataFrame(columns=['gold_clips', 'gold_clips_ans'])
+        clipsList = []
+        for clip in clips:
+            clipUrl = self.make_clip_url(clip)
+            rating = 5
+            if 'noisy' in clipUrl.lower():
+                rating = 1
+
+            clipsList.append({'gold_clips': clipUrl, 'gold_clips_ans': rating})
+
+        df = df.append(clipsList)
+        return df
+
+
+class TrappingSamplesInStore(ClipsInAzureStorageAccount):
+    async def get_dataframe(self):
+        clips = await self.clip_names
+        df = pd.DataFrame(columns=['trapping_clips', 'trapping_ans'])
+        clipsList = []
+        for clip in clips:
+            clipUrl = self.make_clip_url(clip)
+            rating = 0
+            if '_bad_' in clip.lower():
+                rating = 1
+            elif '_poor_' in clip.lower():
+                rating = 2
+            elif '_fair_' in clip.lower():
+                rating = 3
+            elif '_good_' in clip.lower():
+                rating = 4
+            elif '_excellent_' in clip.lower():
+                rating = 5
+
+            clipsList.append(
+                {'trapping_clips': clipUrl, 'trapping_ans': rating})
+
+        df = df.append(clipsList)
+        return df
+
+
+async def prepare_metadata_per_task(cfg, clips, gold, trapping, method, output_dir):
+    """
+    Merge different input files into one dataframe
+    :param test_method
+    :param clips:
+    :param trainings:
+    :param gold:
+    :param trapping:
+    :param general:
+    :return:
+    """
+    df_clips = pd.DataFrame()
+    df_gold = pd.DataFrame()
+    df_trap = pd.DataFrame()
+    metadata_lst = []
+    if clips and os.path.exists(clips):
+        df_clips = pd.read_csv(clips)
+    else:
+        rating_clips_stores = cfg.get(
+            'RatingClips', 'RatingClipsConfigurations').split(',')
+
+        metadata = pd.DataFrame()
+        """
```

Comment:
I'm not sure if this will affect the randomization requirements of P.808. Need to confirm with Scale